src/Account/Session/Session.php

```diff
@@ -33,11 +33,26 @@ abstract class Session implements SessionInterface
     protected $refreshTokenErrors = [];
 
     /**
+     * @deprecated
```

Comment:
in this deprecation notice, you shall mention what public method the user must use instead